### ISSUE TITLE
Download external container file

### ISSUE THREAD

**Issue by cwbsl** [msyk/FMDataAPI#61]
We have big files (200MB and up) that we want to link on a website.
If we use _$file->getContainerData('ContainerFile')_ we get the error `The field 'ContainerFile' is not field name or container field.`.
This piece of code works fine in a database where small images are directly inside the database instead of storing them externally.
How can we receive/download the file from a container where the file is not directly stored inside the database?
Right now my data received via _$file->ContainerFile_ looks like this:

href="file:20180125_151216.gpx\rfilewin:/C:/Users/MyUsername/Downloads/20180125_151216.gpx"

In the Filemaker Server settings under Configuration -> Folder, the "Container Data Folder" is enabled and a folder on my C:\ drive on the server is selected.

**Comment by msyk** [comment]
@cwbsl Thanks for reporting issues! How about your setting in php.ini file? Did your process exceed the limit of main memory?

**Comment by cwbsl** [comment]
No, the memory limit in the php.ini is set to 8GB currently so I guess that's not the issue?!

**Comment by cwbsl** [comment]
Ok great!

**Comment by cwbsl** [comment]
I have also tried to change to the old style as a test with _fmsadmin set serverprefs AuthenticatedStream=2_ but that also didn't make a difference...

**Comment by msyk** [comment]
As far as I checked about FMS 19.5.1, the container field values are started with 'https://...', that is URL to get the content of the file, regardless of whether the storage is internal or external. I've checked with 220MB movie file uploaded by FMDataAPI, and both the secure and open storing. As you mentioned in this first message, the 'file: <filenama>...' is not a valid url because the local file url can't be always valid, ex. the FMS and PHP are different host, so the FMDataAPI is going to raise the exception. Anyway, I couldn't regenerate your situation. I will keep your issue here, and how about wait for the other one's report?
